Mobile Buy SDK/Mobile Buy SDK/Models/Persistent/BUYProduct.m

```diff
@@ -148,12 +148,9 @@ - (BOOL)isDefaultVariant
 {
 	if ([self.variants count] == 1) {
 		BUYProductVariant *productVariant = [self.variants firstObject];
-		BUYOptionValue *optionValue = [productVariant.options anyObject];
-		NSString *defaultTitleString = @"Default Title";
-		NSString *defaultString = @"Default";
-		if ([productVariant.title isEqualToString:defaultTitleString] &&
-			([optionValue.value isEqualToString:defaultTitleString] || [optionValue.value isEqualToString:defaultString])) {
-			return YES;
+		if ([productVariant isDefault]) {
+			BUYOptionValue *optionValue = [productVariant.options anyObject];
```

Comment:
So this method would be reducible to `return (self.variants.count == 1 && self.variants.firstObject.isDefault)`.

We should get used to using property notation.